theme/n21/views/lib/header.hbs

```diff
@@ -7,9 +7,9 @@
                 href="/"
                 aria-label="{{$t 'global.link.landing'}}"
             ><div class="cloud-logo"></div></a>
-            {{#if showAlerts}}
+            {{!-- {{#if showAlerts}} --}}
                 {{> "lib/components/alerts"}}
-            {{/if}}
+            {{!-- {{/if}} --}}
```

Comment:
Why is there commented code here? Are we planning to use it later? If so, maybe add a dev note regarding the purpose.